mission_yaml/SEA70_M27.yml

```diff
@@ -0,0 +1,627 @@
+             Salinity/Density, Conductivity, Density, Salinity"
+  keywords_vocabulary: GCMD Science Keywords
+  metadata_link: http://cfconventions.org/cf-conventions/v1.6.0/cf-conventions.html
+  Metadata_Conventions: CF-1.6, Unidata Dataset Discovery v1.0
+  naming_authority: Voice of the Ocean Foundation
+  platform:    "SeaExplorer Glider"
+  processing_level: "Data provided as is with no expressed or implied
+                     assurance of quality assurance or quality control. L0"
+  publisher_email: callum.rollo@voiceoftheocean.org
+  publisher_name:  Callum Rollo
+  publisher_url:   https://voiceoftheocean.org
+  references:     "created with pyglider https://github.com/c-proof/pyglider"
+  source:     Observational data from a profiling glider.
+  standard_name_vocabulary: CF STandard Name Table v49
+  transmission_system: IRIDIUM
+  glider_name: Scratch
+  glider_serial: "70"
+  wmo_id: "8901002"
+  sea_name: "South Atlantic"
+  comment: 'Calibration date for OCR504 uncertain'
+  contributor_name: Callum Rollo, Louise Biddle, Olle Petersson, Aleksandra Mazur, Marcus Melin
+  contributor_role: Data Processor, PI, Head of Operations, Glider Technician, Glider Technician
+  creator_email: callum.rollo@voiceoftheocean.org
+  creator_name: Callum Rollo
+  creator_url: https://observations.voiceoftheocean.org
+  deployment_id: "27"
+  deployment_name: QUICCHE
+  project: QUICCHE
+  project_url: "https://beal-agulhas.earth.miami.edu/research/projects/quicche/index.html"
+  summary: 'Deployment part of QUICCHE cruise from Cape Town'
+
+
+glider_devices:
+  ctd:
+    make: RBR
+    model: legato
+    serial: '206611'
+    long_name: RBR legato CTD
+    make_model:  RBR legato CTD
+    factory_calibrated: "Yes"
+    calibration_date: "2021-03-17"
+  optics:
+    make: Wetlabs
+    model: FLBBCD
+    serial: '6908'
+    long_name: Wetlabs ECO Puck FLBBCD
+    make_model: Wetlabs FLBBCD
+    factory_calibrated: "Yes"
+    calibration_date: "2021-06-17"
+    calibration_parameters: {Chl_DarkCounts: 49, Chl_SF: 0.0121, CDOM_DarkCounts: 50, CDOM_SF: 0.0906, SIG_DarkCounts: 49, SIG_SF: 3.064E-06}
+  oxygen:
+    make: RBR
+    model: coda
+    serial: '206112'
+    long_name: RBR coda TODO
+    make_model: RBR coda TODO
+    factory_calibrated: "Yes"
+    calibration_date: "2021-03-17"
+  nitrate:
+    make: Seabird
+    model: Deep SUNA
+    serial: 'NTR-1293'
+    long_name: Seabird Deep SUNA
+    make_model: Seabird Deep SUNA
+    factory_calibrated: "Yes"
+    calibration_date: "2019-05-20"
+  irradiance:
+    make: SeaBird
+    model: OCR504
+    serial: '2103'
+    long_name: SeaBird OCR504
+    make_model: SeaBird OCR504
+    factory_calibrated: "Yes"
+    calibration_date: "2021-03-08"
+
+
+# map between glider variables and netcdf variables.  This shouldn't
+# change too much.
+netcdf_variables:
+  keep_variables: ['temperature', 'oxygen_concentration', 'chlorophyll', 'nitrate_concentration', 'down_irradiance_380']
```

Comment:
Shouldn't this be 'downwelling_PAR'

Reply:
As long as it's a variable from the OCR it's fine. Likewise conductivity would work in place of temperature